lib/install/data/data.go

```diff
@@ -68,6 +71,8 @@ type InstallerData struct {
 	KeyPEM  string
 	CertPEM string
 
+	BootstrapISO string
```

Comment:
You don't have to add value in this deprecated structure, cause you have new value in VirtualContainerHost spec.